README.md

```diff
@@ -329,3 +329,9 @@ Build are automatically launched on tagging.
 
 Tags with format image-0.0.0 automatically launch a Docker images build that are available through Docker Hub.
 Tags with format v0.0.0 automatically launch a new release on Github for the TGF executable.
+
+## Update tgf version
```

Comment:
Replace the `## Installation` section instead

Reply:
By **replace**, do you mean remove lines 38 to 59 from the `README.md` file and put my updates instead ?